pkg/skaffold/build/local/local.go

```diff
@@ -45,8 +46,8 @@ func (b *Builder) Build(ctx context.Context, out io.Writer, tags tag.ImageTags,
 	return build.InParallel(ctx, out, tags, artifacts, b.buildArtifact, *b.cfg.Concurrency)
 }
 
-func (b *Builder) buildArtifact(ctx context.Context, out io.Writer, a *latest.Artifact, tag string) (string, error) {
-	digestOrImageID, err := b.runBuildForArtifact(ctx, out, a, tag)
+func (b *Builder) buildArtifact(ctx context.Context, out io.Writer, a *latest.Artifact, opts *build.ImageOptions) (string, error) {
```

Comment:
Since we're passing in the destination image tag, opts shouldn't be a pointer.

Reply:
done